src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -249,6 +249,37 @@ void MockSubProcessCall(Action<Message> message, IRequest request, ISonarLintSet
             statusNotifier.VerifyNoOtherCalls();
         }
 
+        [TestMethod]
+        public async Task TriggerAnalysisAsync_AnalysisFailsDueToInternalMessage_NotifiesOfFailure()
+        {
```

Comment:
New test - if we get an internal rule error message the analysis should be reported as having failed.